.github/workflows/test.yml

```diff
@@ -1,6 +1,13 @@
 name: test
 
-on: [pull_request]
+on:
+  pull_request:
```

Comment:
@strk , is this condition really correct ? Are we going to run (and package and package upload) for each pull request ? We want to upload packages to `dev` only from `master` branch.

Reply:
We have a single workflow here, so it runs BOTH on push and on pull-request.
One step in the workflow (under the "packaging" step) is to DETERMINE publication target.
Possible targets are: 

  - '' (used on pull-request, meaining NO publication)
  - test (used on tag)
  - dev (used on push)

Reply:
See for example https://github.com/linz/linz-lds-bde-schema/runs/708783919?check_suite_focus=true#step:4:19
The run above is for a pull-request, REPO is empty (means no publication)

Reply:
Thanks for clarification.

Reply:
you need to approve this, or I cannot merge